ci/v2/test/core.flaky

```diff
@@ -4,6 +4,7 @@
 //python/ray/tests:test_placement_group_5
 //python/ray/tests:test_runtime_env_2
 //python/ray/tests:test_gcs_fault_tolerance
+//python/ray/tests:test_gcs_ha_e2e
```

Comment:
what is this line for?

Reply:
this test needs a special test environment setup so I keep them in v1 for now